### PR TITLE
add min success design

### PR DESCRIPTION
**Pull request by zen-xu**
Signed-off-by: ZhengYu, Xu <zen-xu@outlook.com>

### CODE REVIEW

File: docs/design/minsuccess.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# MinSuccess
+
+## Introduction
```
Comment (Thor-wl) [inline docs/design/minsuccess.md:3]: Suggest to complete the design doc with more details officially. Please refer to https://github.com/volcano-sh/volcano/blob/master/docs/design/queue-resource-reservation-design.md as example.

**Comment by Thor-wl** [comment]
I'll merge this design first. Please complete the doc anytime you are available, ok? @zen-xu

**Comment by zen-xu** [comment]
> I'll merge this design first. Please complete the doc anytime you are available, ok? @zen-xu

I am completing this design now, so wait a moment

**Review by shinytang6 — APPROVED** [review]
/lgtm

**Pull request merged**